eq/deflect/eventHandler.cpp

```diff
@@ -202,6 +202,12 @@ void EventHandler::_processEvents( const Proxy* proxy )
         case ::deflect::Event::EVT_DOUBLECLICK:
             break;
         case ::deflect::Event::EVT_MOVE:
+             // no break;
+        case ::deflect::Event::EVT_PAN:
+            _proxy->setNavigationMode(
```

Comment:
If the mode is not set, the camera keeps rotating. I'm having a look if the print can be disabled